src/applications/simple-forms/mock-alternate-header-21-0845/components/CustomFormNav.jsx

```diff
@@ -0,0 +1,211 @@
+import React, { useEffect, useState } from 'react';
+import PropTypes from 'prop-types';
+import uniq from 'lodash/uniq';
+
+import {
+  getChaptersLengthDisplay,
+  createFormPageList,
+  createPageList,
+  getActiveExpandedPages,
+  getCurrentChapterDisplay,
+} from 'platform/forms-system/src/js/helpers';
+
+import {
+  focusByOrder,
+  customScrollAndFocus,
+  defaultFocusSelector,
+} from 'platform/utilities/ui';
+
+import { REVIEW_APP_DEFAULT_MESSAGE } from 'platform/forms-system/src/js/constants';
+
+/**
+ * This is a duplicate of FormNav that adds "CHAPTER" above va-segmented-progress-bar
```

Comment:
Note this comment 👀. Most of this file is copy paste

Reply:
Okay cool. Like we've talked about, I think this is a perfectly fine solution for now given that most of this is just testing the waters on the mock header.